### PR TITLE
Balance timeline zooming sensitivity between Firefox and Chrome

### LINKED ISSUE #2768: Time Travel: zoom sensitivity wildly different between Chrome and Firefox
In Firefox, to zoom out from a range of 6 minutes to a range of 1 hour, takes 55(!) full drags of my scroll wheel (one drag is about 8 clicks, so about 440 clicks), which is way too insensitive.

On Chrome, it takes just 3 full drags.

For comparison, zooming by the same 10x factor on Google Maps takes just 13 clicks.

### PR DESCRIPTION
**Pull request by fbarl**
Resolves #2768.

If this will behave nicely, consider extending the same logic to #2787.

### CODE REVIEW

File: client/app/scripts/utils/zoom-utils.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 // See https://github.com/d3/d3-zoom/blob/807f02c7a5fe496fbd08cc3417b62905a8ce95fa/src/zoom.js
 export function defaultWheelDelta(ev) {
-  return ev.deltaY * (ev.deltaMode ? 120 : 1) * 0.002;
+  return ev.deltaY * (ev.deltaMode ? 15 : 1) * 0.0035;
```
Comment (rndstr) [inline client/app/scripts/utils/zoom-utils.js]: Maybe add a comment to elaborate that we are checking `deltaMode` to see whether `deltaY` is in pixel or lines. It seems like only FF uses `1` (`DOM_DELTA_LINE`) while other browsers use `0` (`DOM_DELTA_PIXEL`).

**Comment by rndstr** [comment]
This still has a large discrepancy in sensitivity between FF (lower) and Chromium (higher) for me.

A line-to-pixel ratio in the area of 35 puts it at about the same sensitivity. This would be roughly double what you suggest so maybe there is some fast-scrolling setting in effect for your env (https://stackoverflow.com/a/37474225 suggests that Windows uses 2x).

File: client/app/scripts/utils/zoom-utils.js (hunk 1 above)
Comment (fbarl) [inline client/app/scripts/utils/zoom-utils.js]: Yup, makes sense!

**Comment by fbarl** [comment]
@rndstr After reading about this more, I came to conclusion that any attempt to balance the cross-browser sensitivity perfectly is futile because it will always depend on the platform and some special settings.

Although we seem to be using very similar browsers/environments, line-to-pixel ratio for you seems to be `35px` while for me it's `15px`. I changed it to `25px` in the code and I'd say that if the real sensitivity on browsers will be +/- 50% from that value, that will still result in a good UX.

**Pull request merged**